tb/core/tb_top_verilator.cpp

```diff
@@ -69,7 +69,7 @@ void dump_memory()
 {
     errno = 0;
     std::ofstream mem_file;
-    svLogicVecVal addr = {0};
+    svLogicVecVal addr = {0, 0};
```

Comment:
Why is this necessary?  Is it related to using fusesoc, or is it 'cleanup'?

Reply:
That is a cleanup to make newer GCC happy.